src/Integerations/fhirService.ts

```diff
@@ -0,0 +1,85 @@
+import { HttpService } from '@nestjs/axios';
+import { Injectable } from '@nestjs/common';
+/* eslint-disable */
+/**
+ * FhirService
+ *
+ * This service is responsible for interacting with the FHIR API.
+ */
+@Injectable()
+export class FhirService {
+  constructor(private httpService: HttpService) {}
+  /**
+   * Fetch a specific patient's data from the FHIR API.
+   *
+   * @param {string} nhi - The nhi of the patient.
+   * @returns {Observable} - An Observable that will emit the patient's data when the HTTP request is successful.
+   */
+  getPatientPrescriptionRecords(nhi: string) {
+    // const url = `https://example.com/fhir/MedicationRequest?subject.identifier={NHI}`;
+    // return this.httpService.get(url).pipe(
+    //   map((response) => {
+    //     //Check if this record exist in our DB if not create it in our database
```

Comment:
Please remove this comment, this logic should not be here, we will not do have any operation which will couple our business logic with Fhir service logic.